### PR TITLE
* Added the two booleans `published` and `consumed`

### PR DESCRIPTION
**Pull request by serge-gaia**
* Extract Record filter into a filter class.
* Added some filters

See https://youtrack.gaiaresources.com.au/youtrack/issue/BIOSYS-111

### CODE REVIEW

File: biosys/settings.py [hunk 1]
```diff
@@ -70,7 +71,7 @@
 
 INSTALLED_APPS += PROJECT_APPS
 
-MIDDLEWARE_CLASSES = (
+MIDDLEWARE = [
```
Comment (serge-gaia) [inline biosys/settings.py:74]: new Django 1.10 setting

File: requirements.txt [hunk 2]
```diff
@@ -31,7 +31,7 @@ drf-yasg==1.7.4
 
 
 # dbca for SSO login
-https://static.dbca.wa.gov.au/static/py/dpaw-utils/dist/dpaw-utils-0.3a9.tar.gz
+git+https://github.com/dbca-wa/dpaw-utils.git@0.4.1
```
Comment (tony-gaia) [inline requirements.txt:34]: I take it this means we don't have to ask OIM about this now?

File: biosys/settings.py [hunk 3]
```diff
@@ -24,14 +24,15 @@
 SESSION_COOKIE_SECURE = env('SESSION_COOKIE_SECURE', False)
 if not DEBUG:
     # Localhost, UAT and Production hosts
-    ALLOWED_HOSTS = [
+    ALLOWED_HOSTS = env('ALLOWED_HOSTS',
```
Comment (tony-gaia) [inline biosys/settings.py:27]: :smiley:

File: requirements.txt (hunk 2 above)
Comment (serge-gaia) [reply requirements.txt:34]: No we don't have to for now.

**Pull request merged**